Source/WPEFramework/PluginHost.cpp

```diff
@@ -573,6 +595,11 @@ namespace PluginHost {
             // If we have handlers open up the gates to analyze...
             _dispatcher->Open();
 
+            string id = GetDeviceId(_dispatcher->Services().SubSystemsInterface());
```

Comment:
We are now losing a reference. the returned SubSystemsInterface() is an Addref'ed interface, it is never Released, so it is currently leaking. Move this call (SubSystemsInterface()) into the GetDeviceId method so you can release the interface, if it is given..

Reply:
done